src/v/storage/tests/appender_chunk_manipulations.cc

```diff
@@ -18,7 +18,7 @@
 #include <cstring>
 
 using chunk = storage::segment_appender::chunk;
-static constexpr size_t alignment = 4096;
+static constexpr storage::alignment alignment{4096};
```

Comment:
Should this use `storage::internal::chunk_cache::alignment`?

Reply:
It could, but this PR is just about changing the type rather than the constants